gateway/grpc.go

```diff
@@ -45,18 +44,17 @@ type GrpcGateway struct {
 
 // NewGrpcGateway returns a new gRPC gateway.
 
-func NewGrpcGateway(network config.Network, opts ...grpc.DialOption) (*GrpcGateway, error) {
-	options := []grpc.DialOption{
+func NewGrpcGateway(network config.Network, opts ...grpcAccess.ClientOption) (*GrpcGateway, error) {
+	opts = append(
```

Comment:
This change isn't equivalent to the previous version: Before, the default was insecure, and the options passed to the function were appended, allowing to override this default. Now, the insecure option is appended to the given options, overriding options passed by the user.

Reply:
Good point.

Reply:
are you sure? Isn't it done where the defaults are set first, then any additonal options are appended to override the default?

https://github.com/onflow/flow-go-sdk/blob/master/access/grpc/client.go#L56-L58

Here the default is set as `grpc.WithTransportCredentials(insecure.NewCredentials())`, then any additional options are appended here:

https://github.com/onflow/flow-go-sdk/blob/master/access/grpc/client.go#L66-L70

meaning, shouldn't the final result be

```
dialOptions = []grpc.DialOptions{
grpc.WithTransportCredentials(insecure.NewCredentials()), // from the Default
grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(maxGRPCMessageSize) // added from flowkit
}
```

Reply:
Oh sorry, i just understood what you meant 👍  let me fix this